roles/css-injector/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+all_sar_commands: [ 'sar', 'sarl', 'sarm', 'sarv', 'sarjs' ]
+sarl_commands: [ 'sarl', 'sarm' ]
+sar_fulltext_commands: [ 'sar', 'sarv', 'sarjs' ]
+protocols: [ 'https://', 'http://' ]
+link_attributes:
+  - url\("
+  - url\('
```

Comment:
Ansible seems to handle the escaping of the quotation marks here. I can't think of another attribute besides `url("https://foo.stanford.edu/sites/default/files...`